packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1392,6 +1413,17 @@ export class Bucket extends BucketBase {
     this.inventories.push(inventory);
   }
 
+  private enforceSSL() {
```

Comment:
Expose `enforceSSL` as a property

Reply:
Ok I just removed the enforceSecurityBestPractice property and replaced it with enforceSSL. Thanks for the input! Let me know if there is anything else.